moonsense/pd.py

```diff
@@ -29,7 +28,7 @@ def read_accelerometer(session_file) -> pd.DataFrame:
     df = pd.read_json(session_file, lines=True)
     accelerometer_data = functools.reduce(
         operator.iconcat,
-        df["bundle"].apply(pd.Series)["accelerometer_data"].tolist(),
+        df["bundle"].apply(pd.Series)["accelerometer_data"].dropna().tolist(),
```

Comment:
why is the dropna() needed? Are the bundles returning empty data?

Reply:
I got to see an instance of that. Haven't spent time debugging the root cause yet. We will need to keep an eye on it in the future.